docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
 
-Edits an existing client in the ClientBook.
+**Examples**:
+*  Edit the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+    * `edit 1 p/91234567 e/johndoe@example.com`
+      
+      ![edit 1](images/edit-1.png)
+    
+*  Edit the name of the 2nd person to be `Betsy Crower`.
+    * `edit 2 n/Betsy Crower`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [i/POLICY_NUMBER] [t/TAG]…​`
+[Return to Table of Contents](#table-of-contents)
 
-* Edits the client at the specified INDEX. The index refers to the index number shown in the displayed client list. The index must be a positive integer 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+### List all clients : `list`
+
+**Purpose**: Shows a list of all clients in ClientBook. An optional attribute option can be added to show a list with only the specified attribute.
+
+**Format**: `list [-ATTRIBUTE]`
 
+**Examples**: 
+* Shows a list of all clients and all their information.
+  * `list`
+    
+    ![list](images/list.png)
+    
+*  Shows a list of all clients and their insurance policy number.
+   * `list -i`
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
+    ![list policy](images/list-policy.png)
 
-### Locating persons by name: `find`
+*  Shows a list of all clients and their phone number and insurance policy number.
+   * `list -p -i`
+    
+     ![list phone](images/list-phone-policy.png)
 
-Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
+[Return to Table of Contents](#table-of-contents)
 
-![find_alex_david](images/find_alex_david.png)
+  
+### Search for client contact based on keywords: `find`
 
-![find_alex_&_david](images/find_alex_&_david.png)
+**Purpose**: Finds and displays all clients whose field (name, phone, email, address, tags, insurance policy) contains any of the given keywords.
+An optional attribute option can be added to show the list of matched clients with only the specified attribute.
 
 ![find_alex_&_david_with_filter](images/find_alex_&_david_with_filter.png)
 
-Format: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
+**Format**: `find FLAG/KEYWORD [& MORE_KEYWORDS] [-ATTRIBUTES]…​`
 
-* The search is case-insensitive. e.g hans will match Hans.
-* The order of the keywords does not matter.
+* The search is **case-insensitive**. 
+  * E.g. hans will match Hans.
+* The **order of the keywords does not matter**.
 * Only one `FLAG` can be used in each find command.
 * The `FLAG` can only be from: `n`, `p`, `e`, `a`, `t` and `i`, representing name, phone, email, address, tags and insurance policies respectively.
```

Comment:
Did not state explicitly what the flags are used for. The usage is clear to us as we are the developers. Would it be better to list the flags in a table? Idk

Reply:
* `FLAG` uses the identifier of each attribute to help the program identify which attribute you are referring to.
* For more information about the identifiers for each attribute, refer to [What information can we store for each client contact?](#what-information-can-we-store-for-each-client-contact).
Added the above 2 statements, hope they make things clearer.